cmd/platform.go

```diff
@@ -23,7 +23,8 @@ func configurePlatform(cmd *cobra.Command) {
 	flags.StringP("base-url", "g", "", "Base URL of the target platform, needs to be changed for GitHub enterprise, a self-hosted GitLab instance, Gitea or BitBucket.")
 	flags.BoolP("insecure", "", false, "Insecure controls whether a client verifies the server certificate chain and host name. Used only for Bitbucket server.")
 	flags.StringP("username", "u", "", "The Bitbucket server username.")
-	flags.StringP("token", "T", "", "The personal access token for the targeting platform. Can also be set using the GITHUB_TOKEN/GITLAB_TOKEN/GITEA_TOKEN/BITBUCKET_SERVER_TOKEN/BITBUCKET_CLOUD_APP_PASSWORD environment variable.")
+	flags.StringP("token", "T", "", "The personal access token for the targeting platform. Can also be set using the GITHUB_TOKEN/GITLAB_TOKEN/GITEA_TOKEN/BITBUCKET_SERVER_TOKEN/BITBUCKET_CLOUD_APP_PASSWORD/BITBUCKET_CLOUD_WORKSPACE_TOKEN environment variable.")
+	flags.StringP("authtype", "", "", "The authentication type. Either app-password or workspace-token. Used only for Bitbucket cloud.")
```

Comment:
Having no default option here makes this not backwards compatible for no good reason.

I see three options here:
1. Set the default value to `app-password` to keep backwards compatability
2. Set the default value to `workspace-token` if this is the preferred method of authentication. This would break backwards compatibility, but because of a good reason which could be ok because of the state of the BBC implementation. 
3. The most extreme approach would be to remove password authentication altogether. But if there is no real good reason for people to use it, (meaning they could always use tokens instead,) I would be in favor of this option.

Reply:
Also minor comment on the actual name. Should probably be `auth-type` to match with other names.